packages/react-i18n/src/tests/i18n.test.ts

```diff
@@ -2147,19 +2155,59 @@ describe('I18n', () => {
         expect(sanitizeSpaces(symbol)).toStrictEqual(expectedSymbol);
       },
     );
+
+    it.each`
+      currency | locale     | expectedPrefixed | expectedSymbol
+      ${'BRL'} | ${'en-US'} | ${true}          | ${'R$'}
+      ${'HKD'} | ${'en-US'} | ${true}          | ${'HK$'}
+    `(
+      'returns the listed short symbol in the exceptions list for the given currency - $currency',
+      ({currency, locale, expectedPrefixed, expectedSymbol}) => {
+        const i18n = new I18n(defaultTranslations, {locale});
+        // eslint-disable-next-line dot-notation
+        const {symbol, prefixed} = i18n['getShortCurrencySymbol'](currency);
+        expect(prefixed).toStrictEqual(expectedPrefixed);
+        expect(sanitizeSpaces(symbol)).toStrictEqual(expectedSymbol);
+      },
+    );
   });
 
   describe('#getCurrencySymbol()', () => {
-    it('returns the locale-specific currency symbol and its position', () => {
-      const expected = {
-        symbol: '€',
-        prefixed: true,
-      };
-
-      const i18n = new I18n(defaultTranslations, {locale: 'en'});
+    it.each([
+      {
```

Comment:
Nitpick: For the sake of a more skimmable test, what do you think to using the array notation to avoid a few linebreaks?:

```
it.each([
  ['en', 'EUR', {}, {symbol: '€', prefixed: true}],
  ['fr', 'EUR', {}, {symbol: '€', prefixed: false}],
  ['en', 'SGD', {currencyDisplay: 'narrowSymbol'}, {symbol: 'SGD', prefixed: true}],
  ['en', 'SGD', {}, {symbol: '$', prefixed: true}],
])('returns symbol and position of currency ($currency), given the provided locale ($locale) and options ($options))',
  (locale, currency, options, expected) => {
```